src/apps/loyalty/containers/inquiries/index.tsx

```diff
@@ -84,7 +155,7 @@ interface RelayProps {
     artwork_inquiries_connection: {
       edges: Array<{
         node: {
-          id: string,
+          id: string | null,
```

Comment:
Because MP’s schema doesn’t specify this field as non-nullable https://github.com/artsy/metaphysics/blob/656eb8d1bdaad64ed2fdb0ed2517f8dd0de2ca4a/schema/me/artwork_inquiries.js#L23